bindata/v4.1.0/kube-apiserver/recovery-config.yaml

```diff
@@ -17,6 +17,7 @@ storageConfig:
   ca: /etc/kubernetes/static-pod-resources/etcd-serving-ca-bundle.crt
   urls:
   - "https://localhost:2379"
+servicesSubnet: 172.30.0.0/16
```

Comment:
/cc @squeed

Reply:
NACK, this is a user-configurable range.